shared-ui/components/404/404.styles.ts

```diff
@@ -55,4 +57,113 @@ const ButtonContainer = styled.div`
   }
 `;
 
-export { ButtonContainer, StyledH4, StyledH1 };
+const WaveContainer = styled.div`
+  overflow:hidden;
+  position: absolute;
+`;
+
+const StyledWave1 = styled(motion.img)`
+  padding-top: 25em;
+  display:flex;
+`;
+
+const StyledWave2 = styled(motion.img)`
+  padding-top :4em;
+  padding-left: 30em;
+`;
+
+const StyledWave3 = styled(motion.img)`
+  padding-top :6em;
+  padding-left: 38em;
+`;
+
+const StyledWave4 = styled(motion.img)`
+  padding-top :2em;
+  padding-left: 70em;
+`;
+
+const Penguin = styled.img`
+ bottom : -2em;
+ left: -15em;
+ position: fixed;
+ overflow:hidden;
+ @media ${max.tablet} {
+  scale:0.7;
+  left: -22em;
+  bottom : -7.5em;
+}
+`
+const Boat = styled.img`
+ top: 10em;
+ bottom:30em;
+ float: right;
+ left: 80em;
+ position: fixed;
+ padding : 0;
+ clear: both;
+ overflow:hidden;
+ @media (max-width: 1670px) {
+  left: 70em;
+  }
+  @media (max-width: 1550px) {
+    left: 65em;
+  }
+  @media ${max.desktopLg} {
+    left: 60em;
+  }
+  @media (max-width: 1370px) {
+    left: 55em;
+  }
+  @media ${max.desktop} {
+    left: 50em;
+  }
+  @media (max-width: 1370px) {
+    left: 55em;
+  }
+
+  @media (max-width: 1280px) {
+    left: 50em;
+  }
+
+  @media (max-width: 1190px) {
+    left: 45em;
+  }
+
+  @media (max-width: 1100px) {
```

Comment:
This is a lot of media queries HAHAHAHAH. Instead of setting the left which we will always have to change because the size of the screen changes, let's set the right to be like -5em or something - then it will ALWAYS stick to the right no matter what screen size ur at. 

You can also remove the float: right; and the bottom css style.

The only thing you'd need to change for each screensize is the scale or width and probably top to adjust how far down from the screen it is. But with this approach, you won't need all these extra breakpoints to control the left value. You can just use the built in max.desktop, etc.